README.rst

```diff
@@ -19,7 +18,7 @@ The documentation has five sections:
 - `Getting started <https://rocky.docs.pyansys.com/version/stable/getting_started/index.html>`_: Learn
   how to install PyRocky in user mode and then launch it.
 - `User guide <https://rocky.docs.pyansys.com/version/stable/user_guide/index.html>`_: Understand how to
-  use the Rocky PrePost API with PyRocky.
+  use the Rocky PrePost Scripting with PyRocky.
 - `API reference <https://rocky.docs.pyansys.com/version/stable/api/index.html>`_: Understand PyRocky API
   endpoints, their capabilities, and how to interact with them programmatically.
```

Comment:
```suggestion
- `API reference <https://rocky.docs.pyansys.com/version/stable/api/index.html>`_: Understand how to use
  Python to programmatically interact with PyRocky..
```
The suggested change is due to a new rephrasing as some developers have pointed out the "endpoint" isn't a term used by Python.

Reply:
I'll try to improve that in a following PR.